Library/src/Systems/Spaces/SpaceSystemHelpers.h

```diff
@@ -43,26 +43,27 @@ constexpr const char* SPACE_THUMBNAIL_ASSET_NAME_PREFIX			   = "SPACE_THUMBNAIL_
 
 constexpr const char* AVATAR_THUMBNAIL_ASSET_NAME_PREFIX = "AVATAR_THUMBNAIL";
 
-String GetSpaceMetadataAssetCollectionName(const String& SpaceId);
-Map<String, String> ConvertSpaceMetadataToAssetCollectionMetadata(const String& Metadata);
-String GetSpaceIdFromMetadataAssetCollectionName(const String& MetadataAssetCollectionName);
+csp::common::String GetSpaceMetadataAssetCollectionName(const csp::common::String& SpaceId);
```

Comment:
This file was included in multiple source files, so the `using namespace` statement was also included with it. We typically try to avoid this as it can make namespaces unclear if the developer doesn't know he's importing a `using namespace`. This file went against that pattern and so was refactored.